tainiisanta.js

```diff
@@ -0,0 +1,105 @@
+// есть n человек, необходимо распределить: кто кому дарит подарки
```

Comment:
you divide your function by very small pieces. That's not good. Try to compare it in one function.